src/ModelButtonImage/ModelButtonImage.test.js

```diff
@@ -0,0 +1,50 @@
+import ModelButtonImage from "./ModelButtonImage";
+import ModelButtonSampleImg from "../../static/ModelButtonSampleImg.svg";
+import React from "react";
+import ThemeProvider from "../ThemeProvider";
+import { render } from "@testing-library/react";
+
+// wrapper function for ModelButtonImage
+function ModelImage({ src = ModelButtonSampleImg }) {
+  return <ModelButtonImage src={src} />;
+}
+
+// test suite for ModelButtonImage
+describe("ModelButtonImage", () => {
+  // component should show alt text
+  it("should show alt text  ", () => {
```

Comment:
The test name and test don't match, also i don't think we need to have a test for alt text. Do you mean render component?